core/templates/pages/learner-dashboard-page/card-display/card-display.component.html

```diff
@@ -11,15 +11,11 @@
     </div>
   </div>
   <hr color="#00645c" class="mt-2 mb-4" size="2" width="100%">
-  <div class="d-flex card-display-content-container pb-1 mb-n1" #cards>
+  <div class="d-flex card-display-content-container pb-1 mb-n1" [ngClass]="getVisibility()" #cards>
```

Comment:
Which ng-class are we dynamically adding here?

Reply:
If the component is used in home, an empty string so it displays all the cards and does not apply the flex-wrap class. 

Otherwise, if the width of all cards is > parent div in progress tab, we hide the remaining cards with .card-display-content-hidden and show with .card-display-content-shown.